app/components/Button.tsx

```diff
@@ -19,8 +20,10 @@ export const Button: React.FC<ButtonProps> = ({
   type = 'submit',
   style,
   handleClick = () => {},
+  dataTestId,
```

Comment:
Since this is optional, don't you think we need to provide a default value here?

Reply:
@sumitd94 Kept it optional to not break the previous implementation.